internal/fdv2proto/payloads.go

```diff
@@ -0,0 +1,18 @@
+package fdv2proto
+
+type Payload struct {
+	// The id here doesn't seem to match the state that is included in the
+	// Payload transferred object.
+
+	// It would be nice if we had the same value available in both so we could
+	// use that as the key consistently throughout the the process.
+	ID     string     `json:"id"`
+	Target int        `json:"target"`
```

Comment:
As it currently stands, I think we can probably ignore these fields completely right? The SDK is going to operate under the assumption we will receive a single payload, and that payload is being applied to the entire SDK. So I don't think we actually care that much about the ID and Target fields.